### PR TITLE
Fix adaptive card citation

### PR DESCRIPTION
**Pull request by elopezanaya**
### Description
_Include a description of the problem to be solved_

Miss match  of styles versus copilot tests environment, for self referenced citations, it show the citation as citation-1..Citation-N

image

Additional to this, the style of the text for citations landmark is loss in grey color, 
## Solution Proposed
_Detail what is the solution proposed, include links to design document if required or any other document required to support the solution_

- match citation style for self referenced
- citation label should be the same as the font

### Acceptance criteria
_Define what are the conditions to consider the PR has achieved the intended goal_

## Test cases and evidence
_Include what tests cases were considered, any evidence of testing for future references, to identify any corner cases, etc_

### Link Citatitons

image

### Self referenced
image

### Sanity Tests
-   [ ] You have tested all changes in Popout mode
-   [ ] You have tested all changes in cross browsers i.e Edge, Chrome, Firefox, Safari and mobile devices(iOS and Android)
-   [ ] Your changes are included in the [CHANGELOG](../CHANGE_LOG.md)

## A11y
- [ ] You have run the [Automated Accessibility Check](https://accessibilityinsights.io/docs/en/windows/getstarted/automatedchecks) and have no reported issues

__*Please provide justification if any of the validations has been skipped.*__

### CODE REVIEW

File: chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx [hunk 1]
```diff
@@ -826,11 +826,21 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
...
                 height: .75em;
                 margin-left: .25em;
             }
+
+            .webchat__link-definitions__header-text {
```
Comment (Erli-ms) [inline chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:836]: Is this change on purpose as part of this fix?

Comment (elopezanaya) [reply chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx:836]: yes, to override style of  the label

File: chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/citationsMiddleware.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        return text.replace(/\[(\d+)\]:\s(cite:\d+)\s"([^"]+)"/g, (match, number, citeId) => {
+            const citation = citations.find(c => c.id === citeId);
+            if (citation) {
+                // Replace only the citation label while preserving the original format
+                return `[${number}]: ${citeId} "${citation.title}"`;
+            }
+            return match; // Keep the original match if no replacement is found
+        });
+    } catch (error) {
+
```
[the remark this answers is not in the document]
Comment (elopezanaya) [reply chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/storemiddlewares/citationsMiddleware.ts]: good catch

**Review by mithunjohny — APPROVED** [review]
